### PR TITLE
Cleanup fix - add ability to change AcceleratorName

### PR DESCRIPTION
**Pull request by charliejllewellyn**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: reference-artifacts/Custom-Scripts/SEA-uninstall/README.md [hunk 1]
```diff
@@ -72,7 +72,10 @@ The logic of the script is the following:
 
 3. Before running this script you must manually delete AWS SSO.
 
-4. Execute the script `python3 aws-sea-cleanup.py`
+4. Execute the script `python3 aws-sea-cleanup.py`.
+
+**Note: ** if you used a different AcceleratorName you can use `python3 aws-sea-cleanup.py --AcceleratorName YOUR_ACCELERATOR_NAME`.
```
Comment (Brian969) [inline reference-artifacts/Custom-Scripts/SEA-uninstall/README.md:77]: This should be AcceleratorPrefix (less the -), not Accelerator Name, for consistency :-).

File: reference-artifacts/Custom-Scripts/SEA-uninstall/aws-sea-cleanup.py [hunk 2]
```diff
@@ -910,7 +923,11 @@ def backup_config():
         print("Backing up config.json from CodeCommit...")
         try:
             for repo in repos["repositories"]:
-                if repo["repositoryName"].startswith("PBMM"):
+                if args.AcceleratorName != 'PBMMAccel':
+                    CodeCommitPrefix = args.AcceleratorName
+                else:
+                    CodeCommitPrefix = 'BPMM'
```
Comment (Brian969) [inline reference-artifacts/Custom-Scripts/SEA-uninstall/aws-sea-cleanup.py:929]: PBMM not BPMM :-).

**Comment by Brian969** [comment]
Please resolve merge conflicts and switch to AcceleratorPrefix, instead of Name.

**Review by archikierstead — APPROVED** [review]
/lgtm

**Pull request merged**